ckanext/canada/templates/package/resource_read.html

```diff
@@ -31,6 +31,11 @@
   {% endif %}
 {% endblock %}
 
+{% block breadcrumb_content %}
+  {{ super.super() }}
+  <li class="active"><a href="" title="{{ h.resource_display_name(res) }}">{{ h.resource_display_name(res)|truncate(30) }}</a></li>
```

Comment:
I think the Resource names get pretty long, but we might still be able to support more than `30` chars here?

Reply:
I'll do 80 here as well.